ENIGMAsystem/SHELL/Widget_Systems/DlgMod/dialogs.cpp

```diff
@@ -1,4 +1,4 @@
-/** Copyright (C) 2018 Samuel Venable
+/** Copyright (C) 2019 Samuel Venable

```

Comment:
Generally, leave the original year and use a hyphen (eg, 2018-2019).

Reply:
Yeah, I can't allow this. @time-killer-games you need to use a hyphen, you still own the copyright for 2018.

Reply:
Ok, he fixed it in 3f39481c848855bf927c3c288ceb279baa9831eb